examples/abi.rs

```diff
@@ -60,6 +65,50 @@ async fn run() {
     debug_call!(instance.get_fixed_array());
 }
 
+async fn events(instance: &AbiTypes) {
+    macro_rules! debug_events {
+        (instance.events(). $events:ident ()) => {{
+            let events = instance
+                .events()
+                .$events()
+                .query()
+                .await
+                .expect(concat!(stringify!($events), " failed"));
+            let event_data = events
+                .iter()
+                .map(|event| event.inner_data())
+                .collect::<Vec<_>>();
+            println!("{}()\n  ⏎ {:?}", stringify!($events), event_data);
```

Comment:
Is this a log, or an accidental print statement gone wild?

Reply:
Its a print statement in an example, its not actually part of the library.